src/index.ts

```diff
@@ -33,7 +33,7 @@ export class Ollama {
 
 	constructor (config?: Partial<Config>) {
 		this.config = {
-			address: config?.address ?? "http://127.0.0.1:11434"
+			host: config?.host ?? "http://127.0.0.1:11434"
```

Comment:
Thanks for updating – this is missing a few semantics in https://github.com/jmorganca/ollama-python/commit/43d36489b202904fa554525aaf527ad4bf756bca#diff-0f246a9c084dd2ef9d4a58c02fb818ac4a114c34877e558cac88ab351daae9eeR644

1. Default port `11434`
2. if `http` is specified, port `80`
3. if `https` is specified, port `443`